src/ngraph/runtime/intelgpu/intelgpu_op_custom_kernels.cpp

```diff
@@ -59,6 +59,7 @@ string runtime::intelgpu::get_opencl_type_min_max_value(const element::Type& ngr
     case element::Type_t::u32: return is_min ? "0" : "UINT_MAX";
     case element::Type_t::i16: return is_min ? "SHRT_MIN" : "SHRT_MAX";
     case element::Type_t::u16: return is_min ? "0" : "USHRT_MAX";
+    case element::Type_t::boolean:
     case element::Type_t::i8: return is_min ? "CHAR_MIN" : "CHAR_MAX";
```

Comment:
CHAR_MIN and CHAR_MAX don't seem like the right values for boolean. Even if they are, I don't think it should fall through to i8 and a comment about what is going on is needed.

Reply:
@diyessi According to this https://github.com/NervanaSystems/ngraph/blob/master/src/ngraph/type/element_type.cpp#L66
the nGraph::boolean is the most close to nGraph::i8. Same size, same sign.

Reply:
That's because std::vector<T> is something completely different thing when T is bool than when it is any other type. ngraph bool as two values, 0 and 1.